### PR TITLE
Support L5.5, update spatie/laravel-backup, rename config to backup.php

### LINKED ISSUE #40: Installation on L5.5.1 fails
Here is the console output with all incompatibility details:

```
$ composer require backpack/backupmanager
Using version ^1.2 for backpack/backupmanager
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for backpack/backupmanager ^1.2 -> satisfiable by backpack/backupmanager[1.2.0].
    - Conclusion: remove laravel/framework v5.5.1
    - Conclusion: don't install laravel/framework v5.5.1
    - backpack/backupmanager 1.2.0 requires spatie/laravel-backup ^4.19 -> satisfiable by spatie/laravel-backup[4.19.0, 4.19.1, 4.19.2].
    - spatie/laravel-backup 4.19.0 requires illuminate/notifications ~5.3.28|~5.4.0 -> satisfiable by illuminate/notifications[v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - spatie/laravel-backup 4.19.1 requires illuminate/notifications ~5.3.28|~5.4.0 -> satisfiable by illuminate/notifications[v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - spatie/laravel-backup 4.19.2 requires illuminate/notifications ~5.3.28|~5.4.0 -> satisfiable by illuminate/notifications[v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - don't install illuminate/notifications v5.4.0|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.13|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.17|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.19|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.27|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.36|don't install laravel/framework v5.5.1
    - don't install illuminate/notifications v5.4.9|don't install laravel/framework v5.5.1
    - Installation request for laravel/framework (locked at v5.5.1, required as 5.5.*) -> satisfiable by laravel/framework[v5.5.1].


Installation failed, reverting ./composer.json to its original content.
```

### PR DESCRIPTION
**Pull request by cristianuibar**
Fixes #40

### CODE REVIEW

**Comment by iMokhles** [comment]
@tabacitu @lloy0076 any update when this will be reviewed and merged ?

**Comment by gustavobigardi** [comment]
Is anyone looking at this PR?

File: src/config/backup.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?php
```
Comment (lloy0076) [inline src/config/backup.php]: I guess we can publish this but I wonder if upstream publishes this _as well_?

File: src/config/backup.php [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                'include' => [
+                    base_path(),
+                ],
+
+                /*
+                 * These directories will be excluded from the backup.
+                 * You can specify individual files as well.
+                 */
+                'exclude' => [
+                    base_path('vendor'),
```
Comment (lloy0076) [inline src/config/backup.php]: Why not `node_modules` too? See https://docs.spatie.be/laravel-backup/v5/installation-and-setup

**Comment by lloy0076** [comment]
Nothing I have to say is a major show stopper really - @tabacitu, what say you here?

OTOH this might be a breaking change on an upgrade.

File: src/config/backup.php (hunk 2 above)
Comment (gotrecillo) [reply src/config/backup.php]: I think we could just not publish the config file ourselves.
spatie package has autodiscovery.

we could just add this line to README.

```
php artisan vendor:publish --provider=" Spatie\Backup\BackupServiceProvider"
```

This way we will not have to mantain our config file to mimic theirs.

What do you think?

Comment (lloy0076) [reply src/config/backup.php]: So long as it goes into the documentation for backpack - @tabacitu.

**Comment by cristianuibar** [comment]
Fixed conflicts and corrected the Romanian translation. Can this be merged or another fix be applied? So much time has passed and I'm still waiting for a fix, it's getting frustrating.

**Comment by tabacitu** [comment]
Excellent PR @cristianuibar . Thanks a lot! The backups didn't work for me with the configuration file you provided, but I pulled the new config file from the spatie/backup package and made the appropriate changes - works out of the box now.

I've published this PR as a new point version (1.3.x) so please use [these steps](https://github.com/Laravel-Backpack/BackupManager#upgrading-from-12x-to-13x) if you need to upgrade your existing projects.

Cheers! Keep 'em coming :-)

**Pull request merged**